### DISCUSSION TITLE
【0x04】Set-Purpose v01——尝试在 Ropsten 以太坊测试网上运行 Set-Purpose 的第一个版本

### DISCUSSION THREAD

**Discussion by leeduckgo** [NonceGeek/Move-Camp#12] [Projects]
> Repo：https://github.com/leeduckgo/set-purpose/tree/feat/v0x01-run-on-ropsten

参考指南：

> https://mp.weixin.qq.com/s/cCWNOvkAiJQQkF9Bvpnz_Q

本文阐述了以下玩法：

1. 如何在 Scaffold 中切换各种网络
2. 如何使用submodule在 dApp中切换合约
3. 如何通过github-pages部署我们的dApp

此外，我们在Repo中提供了一键切换 `hardhat network` 和 `react network` 的脚本`changeNetwork.py `。

**本 Project 目标：**

成功通过github-pages部署基于 Ropsten 测试网的 `Set-Purpose` !

**Comment by lysion** [comment]
https://lysion.github.io/set-purpose/

**Comment by gofenix** [comment]
https://zhenfeng-zhu.github.io/set-purpose/

**Reply by gofenix** [reply]
轻服务版本：https://dapp.web.cloudendpoint.cn/

**Reply by leeduckgo** [reply]
> 轻服务版本：https://dapp.web.cloudendpoint.cn/

赞！
